apps/widget-e2e/src/describer/modelAssessment/Constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum locators {
+  IFIPredictionSpan = "span[class^='headerCount']", // IFI - Individual feature importance
+  IFIExpandCollapseButton = "[aria-label='expand collapse group']",
+  IFITableRowSelected = 'div[class^="ms-List-page"] div[class^="ms-DetailsRow"] div[class^="ms-Check is-checked"]',
+  IFIDropdownSelectedOption = "div[class^='featureImportanceChartAndLegend']",
+  IFINumberOfBars = "#FeatureImportanceBar svg .plot .points .point path",
+  IFIYAxisValue = '#FeatureImportanceBar div[class^="rotatedVerticalBox-"]',
+  IFIXAxisValue = '#FeatureImportanceBar g[class^="cartesianlayer"] g[class^="xtick"]',
+  ICEPlot = '#subPlotChoice label:contains("ICE")', // ICE - Individual conditional exception
```

Comment:
tiny nit: it's expectation, not exception, "ICE - Individual conditional exception" -> "Individual Conditional Expectation"
https://christophm.github.io/interpretable-ml-book/ice.html

Reply:
thank you, updated.